config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
   namespace :v1 do
+    mount_devise_token_auth_for "User", at: "auth"
+    resources :sessions, only: [:create, :destroy]
```

Comment:
There's no sessions controller therefore not needed.

Reply:
done